src/components/views/rooms/AuxPanel.js

```diff
@@ -95,9 +95,14 @@ module.exports = React.createClass({
             }
             else {
                 joinText = (<span>
-                    Join as <a onClick={(event)=>{ this.onConferenceNotificationClick(event, 'voice');}}
-                               href="#">voice</a> or <a onClick={(event)=>{ this.onConferenceNotificationClick(event, 'video'); }}
-                               href="#">video</a>.
+                    {_tJsx(
+                        "Join as <a>voice</a> or <a>video</a>.",
```

Comment:
they might want to change the ordering of the strings in the string, plus it's confusing to use the same pattern for both.  I'd use `<a type="voice">` or something similar.